hazelcast/src/main/java/com/hazelcast/nio/tcp/nonblocking/NonBlockingSocketReader.java

```diff
@@ -53,121 +47,24 @@
  * {@link ReadHandler} to get processed.
  */
 public final class NonBlockingSocketReader
```

Comment:
don't you want to use a name as `MemberNonBlockingSockerReader` ? something what makes this subclass unique.